src/site/layouts/health_care_region_health_services_page.drupal.liquid

```diff
@@ -17,202 +17,20 @@
-                            {% endfor %}
-                        </ul>
-                    </section>
-                {% endif %}
-
-                {% if primaryCareHealthServices.entities.length %}
-                    <section class="primary-health-services" id="primary-health-services">
-                        <h2>Primary care</h2>
-                        {% for primaryService in primaryCareHealthServices.entities %}
-                            {% include "src/site/facilities/health_service.drupal.liquid" with healthService = primaryService %}
-                        {% endfor %}
-                    </section>
-                {% endif %}
-
-                {% if extendedCareHealthServices.entities.length %}
-                    <section class="extended-care-health-services" id="extended-care-health-services">
-                        <h2>Extended care and rehabilitation</h2>
-                        {% for service in extendedCareHealthServices.entities %}
-                            {% include "src/site/facilities/health_service.drupal.liquid" with healthService = service %}
-                        {% endfor %}
-                    </section>
-                {% endif %}
-
-                {% if homelessHealthServices.entities.length %}
-                    <section class="homeless-health-services" id="homeless-health-services">
-                        <h2>Homeless services</h2>
-                        {% for service in homelessHealthServices.entities %}
-                            {% include "src/site/facilities/health_service.drupal.liquid" with healthService = service %}
-                        {% endfor %}
-                    </section>
-                {% endif %}
-
-                {% if socialProgramsPatientFamilyServices.entities.length %}
-                    <section class="social-patient-family-services" id="social-patient-family-services">
-                        <h2>Social programs and services</h2>
-                        {% for socialService in socialProgramsPatientFamilyServices.entities %}
-                            {% include "src/site/facilities/health_service.drupal.liquid" with healthService = socialService %}
-                        {% endfor %}
-                    </section>
-                {% endif %}
-
-                {% if genomicMedicineHealthServices.entities.length %}
-                    <section class="genomic-medicine-health-services" id="genomic-medicine-health-services">
-                        <h2>Genomic medicine/medical genetics (Genetic medicine)s</h2>
-                        {% for service in genomicMedicineHealthServices.entities %}
+                <section id="table-of-contents">
+                    <h2 class="vads-u-margin-bottom--2 vads-u-font-size--lg">On this page</h2>
+                    <ul class="usa-unstyled-list"></ul>
```

Comment:
just wanted to point out that this is empty. Don't understand liquid templates completely yet so this might be intentional.

Reply:
yes, this gets filled in with the build - when a certain metalsmith plugin runs - based on the `<h2>` tags on the page :smile: